v2/emailpassword/custom-ui/init/user-management-dashboard/setup.mdx

```diff
@@ -0,0 +1,251 @@
+---
+id: setup
+title: "Setting up the dashboard"
+hide_title: true
+---
+
+# Setting up the Dashboard
+
+<!-- COPY SECTION -->
+<!-- ./userdashboard/about.mdx -->
+<!-- 1 -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
+import AppInfoForm from "/src/components/appInfoForm"
+import CustomAdmonition from "/src/components/customAdmonition"
+
```

Comment:
Add a section here about the dashboard architecture. Essentially copy this part: https://supertokens.com/docs/thirdpartyemailpassword/introduction#architecture, with the image and everything. But you don't need to explain frontend and backend sdk and all. Just say that the dashboard is served by the backend SDK.

Then also change the image in this section (https://supertokens.com/docs/thirdpartyemailpassword/introduction#architecture) to be without dashboard in it, and remove the "Serves the user management dashboard which can be accessed on the /auth/dashboard path of your API domain." point from it. Make sure that this is done across all recipes.

Reply:
done